model_api/python/model_api/models/ssd.py

```diff
@@ -176,5 +176,6 @@ def __call__(self, outputs):
         return detections
 
 
+_bbox_area_threshold = 1.0
 _saliency_map_name = "saliency_map"
 _feature_vector_name = "feature_vector"
```

Comment:
How about setting these values as attribute or class attirbute?

Reply:
This will be discussed during MAPI refactoring round